package.json

```diff
@@ -12,77 +12,56 @@
     "end:test": "node_modules/gulp/bin/gulp.js kill-serverless",
     "debug": "export SLS_DEBUG=* && node --inspect ./node_modules/serverless/bin/serverless offline start",
     "test": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.unitTest.ts\" --coverage",
-    "test-i": "BRANCH=local jest --testMatch=\"**/*.intTest.ts\" ",
+    "test-i": "BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand",
     "test:dev": "npm run build:dev && npm run test",
     "prepush": "npm test && npm run tslint && npm run security-checks",
     "predeploy": "npm install && npm run prepush",
     "security-checks": "git secrets --scan && git log -p | scanrepo",
     "tslint": "tslint tests/**/*.ts src/**/*.ts --fix",
     "sonar-scanner": "npm run tslint && npm run coverage && sonar-scanner",
-    "coverage": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.*Test.ts\" --coverage",
+    "coverage": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.*Test.ts\" --coverage --runInBand",
     "audit-dep": "npm audit --json | audit-filter --nsp-config=.nsprc --audit=-"
   },
-  "nyc": {
-    "include": [
-      "src/**/*.ts",
-      "src/**/*.tsx"
-    ],
-    "extension": [
-      ".ts",
-      ".tsx"
-    ],
-    "require": [
-      "ts-node/register"
-    ],
-    "reporter": [
-      "text-summary",
-      "html"
-    ],
-    "sourceMap": true,
-    "instrument": true
-  },
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "aws-sdk": "^2.473.0",
-    "aws-xray-sdk": "^2.3.3",
+    "aws-sdk": "^2.560.0",
+    "aws-xray-sdk": "^2.4.0",
     "node-yaml": "^3.2.0",
     "path-parser": "^4.2.0",
     "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
-    "@types/chai": "^4.1.7",
-    "@types/jest": "^24.0.15",
+    "@types/jest": "^24.0.21",
     "@types/jest-plugin-context": "^2.9.2",
     "@types/lambda-tester": "^3.6.0",
-    "@types/node": "^12.6.8",
-    "@types/request": "^2.48.1",
-    "@types/sinon": "^7.0.13",
+    "@types/node": "^12.12.3",
+    "@types/request": "^2.48.3",
+    "@types/sinon": "^7.5.0",
     "@types/supertest": "^2.0.8",
-    "audit-filter": "^0.4.3",
+    "audit-filter": "^0.5.0",
     "aws-lambda-mock-context": "^3.2.1",
-    "chai": "^4.2.0",
-    "expect": "^24.8.0",
+    "expect": "^24.9.0",
     "gulp": "^4.0.2",
+    "husky": "^3.0.9",
     "jest": "^24.8.0",
     "jest-plugin-context": "^2.9.0",
     "jest-sonar-reporter": "^2.0.0",
-    "lambda-tester": "^3.5.0",
-    "nyc": "^14.1.1",
-    "serverless": "^1.45.0",
+    "lambda-tester": "^3.6.0",
+    "serverless": "^1.56.1",
     "serverless-dynamodb-local": "^0.2.37",
-    "serverless-offline": "^3.33.0",
+    "serverless-offline": "^5.12.0",
     "serverless-plugin-tracing": "^2.0.0",
     "serverless-plugin-typescript": "^1.1.5",
-    "sinon": "^7.4.1",
+    "sinon": "^7.5.0",
```

Comment:
this can be removed

Reply:
No it can't. Bhavana didn't get rid of ALL instances

Reply:
And I don't have time to do it now

Reply:
okay in that case can we create a ticket so we may keep track of this and can re-visit later

Reply:
created tech debt ticket https://jira.dvsacloud.uk/browse/CVSB-9496 to capture revisiting Sinon

Reply:
created https://jira.dvsacloud.uk/browse/CVSB-9496 to do this